scripts/create-component/plop-templates-component/src/components/{{componentName}}/use{{componentName}}.ts.hbs

```diff
@@ -1,35 +1,30 @@
 import * as React from 'react';
 import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utils';
+import { useMergedRefs } from '@fluentui/react-hooks';
 import { {{componentName}}Props, {{componentName}}State } from './{{componentName}}.types';
-import { use{{componentName}}State } from './use{{componentName}}State';
-
 
 /**
  * Consts listing which props are shorthand props.
  */
-export const {{camelCase componentName}}ShorthandProps = ['loader', 'content'];
+export const {{camelCase componentName}}ShorthandProps = [];
 
-const mergeProps = makeMergeProps({ deepMerge: {{camelCase componentName}}ShorthandProps });
+const mergeProps = makeMergeProps<{{componentName}}State>({ deepMerge: {{camelCase componentName}}ShorthandProps });
 
 /**
- * Given user props, returns state and render function for a {{componentName}}.
+ * Returns the props and state required to render the component
```

Comment:
It might be good to add `@param` annotations explaining the source of `props` and `ref` (this is kind of what the old "given user props" wording did)